web/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM nginx:alpine
+
+LABEL varion="1.0" \
+    maintainer="Markus Hadenfeldt <docker@teaspeak.de>, h1dden-da3m0n" \
+    description="A simple TeaSpeak WebClinet running on Nginx-Alpine"
```

Comment:
`WebClinet` I guess there is a type (may be even from me :D)

Reply:
I think that was my doing, fixed anyway ;-)